vllm/assets/service_checks.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "agent_version": "7.56.0",
+        "integration": "vLLM",
+        "check": "vllm.openmetrics.health",
+        "statuses": [
+            "ok",
+            "critical"
+        ],
+        "groups": [
+            "host",
+            "endpoint"
+        ],
+        "name": "vLLM OpenMetrics endpoint health",
+        "description": "Returns `CRITICAL` if the Agent is unable to connect to the vLLM OpenMetrics endpoint, otherwise returns `OK`."
+    },
+    {
+        "agent_version": "7.56.0",
+        "integration": "vLLM",
+        "check": "vllm.health.status",
```

Comment:
Hmm, I thought we weren't adding service checks that go beyond the built-in OpenMetrics one. It looks like the RFC only mentions that service check too.

Reply:
Yea I found this `health` endpoint a bit later and figured I'd add it as another metric that we can collect since we're hitting their API anyway